### PR TITLE
[Outlook] (OCS) Add supported build

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[d385efa](https://github.com/OfficeDev/office-js-docs-pr/commits/d385efa3a0da832197b9e92dae594178fae041aa)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/outlook/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-build/docs/outlook/privacy-and-security.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/privacy-and-security?branch=pr-en-us-2249) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C10%5C5%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202010051906192058-2249%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=YzLqbD9TAiPqz5nJizoVP9uxHJjO0dkmJpU1l97Y7OM%3D&st=2020-10-05T19%3A02%3A00Z&se=2020-11-05T19%3A07%3A00Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/outlook/privacy-and-security.md [hunk 1]
```diff
@@ -56,12 +56,14 @@ The following figure shows the four levels of permissions and describes the capa
...
+End users and IT admins can turn off [optional connected experiences in Office](/deployoffice/privacy/optional-connected-experiences) desktop and mobile clients. For Outlook add-ins, the impact of disabling the **Optional connected experiences** setting depends on the client but usually means that user-installed add-ins and access to the Office Store are not allowed. Certain Microsoft add-ins that are considered essential or business-critical, and add-ins deployed by an organization's IT admin through [Centralized Deployment](../publish/centralized-deployment.md) will still be available.
 
-- Windows, Mac: The **Get Add-ins** button is not displayed so users can no longer manage their add-ins or access the Office Store.
+- Windows\*, Mac: The **Get Add-ins** button is not displayed so users can no longer manage their add-ins or access the Office Store.
 - Android, iOS: The **Get Add-ins** dialog shows only admin-deployed add-ins.
 - Browser: Availability of add-ins and access to the Store are unaffected so users can continue to [manage their add-ins](https://support.microsoft.com/office/8f2ce816-5df4-44a5-958c-f7f9d6dabdce), including admin-deployed ones.
```
Comment (JuaneloJuanelo) [inline docs/outlook/privacy-and-security.md:64]: the support described above

**Review by JuaneloJuanelo — CHANGES_REQUESTED** [review]

**Comment by o365devx** [comment]
Docs Build status updates of commit _[ced03f6](https://github.com/OfficeDev/office-js-docs-pr/commits/ced03f6e9e0132e9d61bca7e3fe0ca0224f03bd1)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/outlook/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-build/docs/outlook/privacy-and-security.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/privacy-and-security?branch=pr-en-us-2249) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C10%5C5%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202010052020403991-2249%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=pI9a9KD5r7jicYfqkKLj1korKX8lYJiwfL11cYRdzz0%3D&st=2020-10-05T20%3A16%3A26Z&se=2020-11-05T20%3A21%3A26Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[4f3360c](https://github.com/OfficeDev/office-js-docs-pr/commits/4f3360c072ca682ff83503c80321747a83916c5b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/outlook/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-build/docs/outlook/privacy-and-security.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/privacy-and-security?branch=pr-en-us-2249) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C10%5C5%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202010052027381633-2249%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=VkGI%2BeyaXTg4N0M4Q6R5ilBvQvoXDJoPBGuM%2FiCM%2Fb0%3D&st=2020-10-05T20%3A23%3A12Z&se=2020-11-05T20%3A28%3A12Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[ac8e0ce](https://github.com/OfficeDev/office-js-docs-pr/commits/ac8e0cef5dcc0ce91a51a591dc95273fdb5bdd3a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/outlook/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-build/docs/outlook/privacy-and-security.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/privacy-and-security?branch=pr-en-us-2249) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C10%5C5%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202010052031034042-2249%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=2CCFxXWKEjp7icpDB0GxMj%2Bri4SQjOLKx%2BxO85%2FD3%2Bk%3D&st=2020-10-05T20%3A26%3A43Z&se=2020-11-05T20%3A31%3A43Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[dd4f526](https://github.com/OfficeDev/office-js-docs-pr/commits/dd4f52638dc823d54438870e43b9b0ed738caea8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/links-get-office-versions-builds.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-build/docs/includes/links-get-office-versions-builds.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/add-in-commands-requirement-sets?branch=pr-en-us-2249) |
[docs/outlook/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-build/docs/outlook/privacy-and-security.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/privacy-and-security?branch=pr-en-us-2249) |
[docs/reference/requirement-sets/outlook-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-build/docs/reference/requirement-sets/outlook-api-requirement-sets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/outlook-api-requirement-sets?branch=pr-en-us-2249) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C10%5C6%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202010061552571169-2249%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=f0ibpr%2B0HmW2yitgjODKcXOD%2BJXgMD3fmgaojAVKqqc%3D&st=2020-10-06T15%3A48%3A32Z&se=2020-11-06T15%3A53%3A32Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**